2016-07/july-31.md

```diff
@@ -0,0 +1,141 @@
+
+This wasn't anything bad on [Phabricator's](https://www.phacility.com/) part: it provided a lot of benefits including being able to specify custom fields so users could input their Babel version, `.babelrc` config, as well as input/output code.
+
+Many people have said that they haven't submitted issues to Babel (whether it be bugs or not) because of the unfamiliarity of phabricator. Most are expecting github issues (especially since the rest of our repo and pull requests are there). Instead what we got were more people reporting things through the [website repo](https://github.com/babel/babel.github.io) about babel itself, commenting on commits, etc. When you are trying to get new contributors when talking at meetups or open source events, it just becomes another barrier to contribution. Github has also recently added some features for maintainers.
+
+#### Plan
+
+Migrate updates to existing issues and create new issues for issues started on Phabricator using github's issue API.
+
+@danez has done a great job getting this effort started:  https://github.com/babel/phabricator-to-github. There's also a [TODO issue](https://github.com/babel/phabricator-to-github/issues/2).
+
+### Preset Options ([PR #3331](https://github.com/babel/babel/pull/3331))
+
+[There are babel presets](https://www.npmjs.com/search?q=babel-preset-) for like every version of node, browser and different combinations for webpack, loose mode, etc.
+
+I think it's safe to say we need something like this now? We need to decide what we want a preset to be able to do. Should it allow blacklisting (people ask about this a lot) or anything a user wants? Or just provide the common use cases as options. Many people are confused/frustrated about presets/configuration so this should help a bit.
+
+After this change, we can modify the es2015 preset to provide options to turn off the commonjs transform and to turn on loose mode (very commonly asked). This would also be a prereq to making a preset that acts as a "autoprefixer" for babel [#3476](https://github.com/babel/babel/pull/3476).
+
+### Codemods ([PR #3561](https://github.com/babel/babel/pull/3561))
+
+Just have to modify the PR to resolve a module that acts as the babel parser/generator so we can use something like recast to generate code that retains formatting. Ref [mohebifar/lebab#138](https://github.com/mohebifar/lebab/issues/138)
+
+### `babel-preset-latest`
+
+We should probably do another [poll](https://twitter.com/left_pad/status/758429846594850816) for the name...
+
+Given the es20xx presets only transform each year’s transforms, I think many would appreciate a way to include everything that has been standardized so far. This includes everything in stage-4 and everything es2015+. This wouldn't be difficult to add.
+
+So instead of doing:
+
+```js
+{ "presets": ["es2015", "es2016", "es2017", ...] }
+```
+
+```js
+{ "presets": ["latest"] }
```

Comment:
Someone can make a PR for this as well! It would basically just include `"es2015", "es2016", "es2017"` and we'll continue to update it